doc/doc.sh

```diff
@@ -20,10 +20,11 @@ fi
 
 case $1 in
   prepare)
-    rm -rf framework
+		if [[ -n "$CI" ]]; then
```

Comment:
unconsitent test style. Please choose btw simple or double braquets.

Reply:
Double braquets is a built-in Bash syntax not compatible with SH. Those files are executed using /bin/sh:
* https://github.com/kuzzleio/kuzzle/pull/1394/files/425a54fbc4092ee8b8b78593f069d9d72cb624e6#diff-2b48d1e4ced50319e0006fe5fa27a50c
* https://github.com/kuzzleio/kuzzle/pull/1394/files/425a54fbc4092ee8b8b78593f069d9d72cb624e6#diff-d81051915dfd25ff81fc2f7ccbf0132e

So I use the simple braquet syntax to avoid warnings.